proposals/zips/zip005.sol

```diff
@@ -0,0 +1,219 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import {Addresses} from "@proposals/Addresses.sol";
+import {Proposal} from "@proposals/proposalTypes/Proposal.sol";
+import {TimelockProposal} from "@proposals/proposalTypes/TimelockProposal.sol";
+import {ERC1155MaxSupplyMintable} from "@protocol/nfts/ERC1155MaxSupplyMintable.sol";
+
+contract zip005 is Proposal, TimelockProposal {
+    string public name = "ZIP005";
+    string public description = "ZTX CGv1.2 MaxSupply updates for placeables";
+
+    struct TokenIDMaxSupplySettings {
+        uint256 tokenId;
+        uint256 maxSupply;
+    }
+
+    TokenIDMaxSupplySettings[] public placeableTokenIDMaxSupplySettings;
+
+    function setAndConfirmPlaceableData() public {
```

Comment:
Question: We could pass an array of structures. This would make it easier to update. Also, why is this public? I would assume it would be restricted to something like Roles.ADMIN. However, based on prior commits, this looks good.